public/app/plugins/datasource/cloudwatch/components/MetricStatEditor/MetricStatEditor.tsx

```diff
@@ -32,6 +32,9 @@ export function MetricStatEditor({
   const metrics = useMetrics(datasource, metricStat);
   const dimensionKeys = useDimensionKeys(datasource, { ...metricStat, dimensionFilters: metricStat.dimensions });
   const accountState = useAccountOptions(datasource.resources, metricStat.region);
+  const percentileSyntaxRE = /^(p|tm|tc|ts|wm)\d{2}(?:\.\d{1,2})?$/;
+  const boundariesInnerParenthesesSyntax = `\\d*(\\.\\d+)?%?:\\d*(\\.\\d+)?%?`;
+  const boundariesSyntaxRE = new RegExp(`^(PR|TM|TC|TS|WM)\\((${boundariesInnerParenthesesSyntax})\\)$`);
```

Comment:
The regexes can be moved outside of the component.

It can be moved above the line `export function MetricStatEditor({`